app/models/scholarship_application.rb

```diff
@@ -0,0 +1,15 @@
+class ScholarshipApplication < ApplicationRecord
+  validate :user_is_verified
+
+  belongs_to :user
+  belongs_to :scholarship
+
+  delegate :name, to: :user
+  delegate :email, to: :user
+
+  def user_is_verified
+    unless self.user.verified
```

Comment:
A style option, nothing major.  For these `boolean` attributes, you could also end them with a `?`, and it will do the same thing; however, it sort of grammatically appears as if you are asking a yes/no question.   For example:

```ruby
  unless self.user.verified?
```